peps/pep-0763.rst

```diff
@@ -264,6 +269,115 @@ usage, this PEP identifies several limitations to this approach:
    PyPI is not straightforward, and there is limited possibility to gather a
    project's download statistics from mirrors or other distributions systems.
 
+.. _appendix-a:
```

Comment:
We sometimes prefix the PEP number so these are unique across the whole Sphinx build and prevent things like:

`/home/runner/work/peps/peps/peps/pep-0763.rst:275: WARNING: duplicate label appendix-a, other instance in /home/runner/work/peps/peps/peps/pep-0753.rst`

```suggestion
.. _763-appendix-a:
```

Reply:
Whoops, realized after I pushed. Fixed!